README.md

```diff
@@ -0,0 +1,3 @@
+node-v10.16.3
+React version: 16.11.0
+npm -v 6.9.0
```

Comment:
also please take your attention that it could be misleading for newcomer how we should start project?
using docker-compose or "scripts" from package.json file